lib/assert.js

```diff
@@ -26,6 +26,8 @@ const util = require('util');
 const Buffer = require('buffer').Buffer;
 const pToString = (obj) => Object.prototype.toString.call(obj);
 
+const trace_mgr = require('trace_mgr'); //ENABLE_TTD
```

Comment:
I wonder if this is something that would be problematic if and when we want to merge back to mainline node- for now I suppose it's fine, but we should keep track of this somewhere- ideally, it would be nice if trace_mgr is used only in a ttd-enabled build of node